README.md

```diff
@@ -1,115 +1,82 @@
-  data retrieval.
-* Mutating JSON: changing the contents of a JSON document can be done using the
-  [sjson](https://github.com/tidwall/sjson) library.
-* Generic `set` implementation: using [Set](https://en.wikipedia.org/wiki/Set_(abstract_data_type))
-  data types can significantly reduce the amount of code inside of a policy,
-  see the `union`, `intersection`, `difference`,... operations provided
-  by a Set implementation.
-  The [mapset](https://github.com/deckarep/golang-set) can be used when writing
-  policies.
+### Example
 
-Last but not least, this policy takes advantage of helper functions provided
-by [Kubewarden's Go SDK](https://github.com/kubewarden/policy-sdk-go).
+With this policy deployed and configured as:
 
-## Testing
-
-This policy comes with a set of unit tests implemented using the Go testing
-framework.
-
-As usual, the tests are defined inside of the `_test.go` files. Given these
-tests are not part of the final WebAssembly binary, the official Go compiler
-can be used to run them. Hence they can take advantage of the `encoding/json`
-package to reduce some testing boiler plate.
-
-The unit tests can be run via a simple command:
-
-```shell
-make test
+``` yaml
+forbiddenSysctls:
+- net.ipv6.conf.lo.*
+allowedUnsafeSysctls:
+- net.ipv6.conf.lo.max_addresses
 ```
 
-It's also important the test the final result of the TinyGo compilation:
-the actual WebAssembly module.
-
-This is done by a second set of end-to-end tests. These tests use the
-`policicy-testdrive` cli provided by the Kubewarden project to load and execute
-the policy.
 
-The e2e tests are implemented using [bats](https://github.com/sstephenson/bats):
-the Bash Automated Testing System.
-
-The end-to-end tests are defined inside of the `e2e.bats` file and can
-be run via this commmand:
-
-```shell
-make e2e-tests
+A pod specifying the following Sysctls would get permitted, as they are
+allowedUnsafe or on the default safe set:
+
+``` yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sysctl-example
+spec:
+  securityContext:
+    sysctls:
+    - net.ipv6.conf.lo.max_addresses
+      value: "1024"
+    - name: kernel.shm_rmid_forced
+      value: "0"
+...
 ```
 
-## Automation
 
-This project contains the following [GitHub Actions](https://docs.github.com/en/actions):
-
-  * `e2e-tests`: this action builds the WebAssembly policy, installs
-    the `bats` utility and then runs the end-to-end test
-  * `unit-tests`: this action runs the Go unit tests
-  * `release`: this action builds the WebAssembly policy and pushes it to a
-    user defined OCI registry ([ghcr](https://ghcr.io) is a perfect candidate)
+Yet the following pod will get rejected, as `net.ipv6.conf.lo.mtu` is forbidden,
+even if `kernel.shm_rmid_forced` is part of the default safe set:
+
+``` yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sysctl-example
+spec:
+  securityContext:
+    sysctls:
+    - name: kernel.shm_rmid_forced
+      value: "0"
+    - net.ipv6.conf.lo.mtu
```

Comment:
```suggestion
    - name: net.ipv6.conf.lo.mtu
```

Reply:
doh.. fixed in https://github.com/kubewarden/psp-sysctl-policy/pull/1/commits/729fcd8e9e3deeb5c0b8010ff27f55d34c4acc91.